### PR TITLE
Improve users tab in projects page

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/views/Project/Details/Users/SearchList/SearchListItem/index.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+@connect(undefined, mapDispatchToProps)
+export default class SearchListItem extends React.PureComponent {
+    static propTypes = propTypes;
+    static defaultProps = defaultProps;
+
+
+    constructor(props) {
+        super(props);
+
+        const setState = d => d.setState(d);
```
Comment (tnagorra) [inline src/views/Project/Details/Users/SearchList/SearchListItem/index.js:48]: what does this even mean? d.setState(d)

File: src/views/Project/Details/Users/SearchList/SearchListItem/index.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                <div className={className}>
+                    <div className={iconClassName} />
+                    <div className={styles.title}>
+                        { usergroupTitle }
+                    </div>
+                    <div className={styles.actionButtons}>
+                        <PrimaryButton
+                            onClick={this.handleAddUsergroupButtonClick}
+                            iconName={iconNames.add}
+                            title="Add usergroup to the project"
```
Comment (tnagorra) [inline src/views/Project/Details/Users/SearchList/SearchListItem/index.js:127]: Translation should be done for every string

File: src/views/Project/Details/Users/SearchList/SearchListItem/index.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                <div className={className}>
+                    <div className={iconClassName} />
+                    <div className={styles.title}>
+                        { username }
+                    </div>
+                    <div className={styles.actionButtons}>
+                        <PrimaryButton
+                            onClick={this.handleAdduserButtonClick}
+                            iconName={iconNames.add}
+                            title="Add user to the project"
```
Comment (tnagorra) [inline src/views/Project/Details/Users/SearchList/SearchListItem/index.js:155]: No translation detected. This is an automated code review.

File: src/views/Project/Details/Users/SearchList/SearchListItem/requests/UserMembershipRequest.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    handleAfterLoad = () => {
+        this.parent.setState({ pendingMembershipRequest: false });
+    }
+
+    handleSuccess = (response) => {
+        const {
+            results = [],
+        } = response;
+
+        console.warn(results);
```
Comment (tnagorra) [inline src/views/Project/Details/Users/SearchList/SearchListItem/requests/UserMembershipRequest.js:23]: console.warn are very harmful for humans

File: src/views/Project/Details/Users/SearchList/index.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+        return;
+    }
+
+    searchRequest
+        .init(searchText)
+        .start();
+});
+
+const EmptySearch = () => {
+    const emptyText = 'No matching user / usergroup found';
```
Comment (tnagorra) [inline src/views/Project/Details/Users/SearchList/index.js:44]: No translation detected. This is an automated code review.

File: src/views/Project/Details/Users/SearchList/index.js [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    return (
+        <div className={styles.emptyText}>
+            { emptyText }
+        </div>
+    );
+};
+
+const SearchTip = () => {
+    const tipText = 'Search with at least 3 characters';
```
Comment (tnagorra) [inline src/views/Project/Details/Users/SearchList/index.js:53]: No translation detected. This is an automated code review.

File: src/views/Project/Details/Users/SearchList/SearchListItem/index.js (hunk 1 above)
Comment (frozenhelium) [reply src/views/Project/Details/Users/SearchList/SearchListItem/index.js:48]: LOL, this should be 
d => this.setState(d)

**Pull request closed without merging**